react/features/speaker-stats/components/AbstractSpeakerStatsList.js

```diff
@@ -61,22 +60,30 @@ const abstractSpeakerStatsList = (speakerStatsItem: Function, isWeb: boolean = t
         return stats;
     });
 
+    const [ state, setState ] = useState(getLocalSpeakerStats());
```

Comment:
there is already a state of the stats kept in the speaker stats reducer which is also set by the search criteria